### PR TITLE
preparation for viber implementation, unification messaging logic 

### PR DESCRIPTION
**Pull request by ibilous**
What is intended to be done here:
1) Preparation for implementation viber messaging (in LEP).  `CustomCommunicationMessageHandler` is responsible for this
2) Unification messaging logic for kafka consumer and rest request

The main concern i can see in my implementation is KafkaChannelFactory class. The problem i faced with was about putting tenant to the tenant thread context for further lep call. I have managed to deal with it but not sure that it is the best way. Could u advice me pls ?

### CODE REVIEW

File: src/test/java/com/icthh/xm/tmf/ms/communication/config/SmppTest.java [hunk 1]
```diff
@@ -54,13 +54,4 @@ public void testSendingSms() {
         assertNotNull(messageId);
     }
 
-    @Test
```
Comment (ishkurko) [inline src/test/java/com/icthh/xm/tmf/ms/communication/config/SmppTest.java:57]: Why test removed?

Comment (ibilous) [reply src/test/java/com/icthh/xm/tmf/ms/communication/config/SmppTest.java:57]: coz i have removed the method this test tested

File: src/main/java/com/icthh/xm/tmf/ms/communication/messaging/handler/SmppMessagingHandler.java [hunk 2]
```diff
@@ -77,11 +72,64 @@ public void receiveMessage(CommunicationMessage message) {
         }
     }
 
+    @Override
+    public void handle(CommunicationMessageCreate messageCreate) {
+        CommunicationMessage message = new CommunicationMessage();
```
Comment (sergeysenja1992) [inline src/main/java/com/icthh/xm/tmf/ms/communication/messaging/handler/SmppMessagingHandler.java:77]: how about use mapstruct ?

Comment (ibilous) [reply src/main/java/com/icthh/xm/tmf/ms/communication/messaging/handler/SmppMessagingHandler.java:77]: ok

**Pull request merged**